src/objects/RecurringDonationSchedule__c.object

```diff
@@ -0,0 +1,485 @@
+                    <fullName>1st and 15th</fullName>
+                    <default>false</default>
+                    <label>1st and 15th</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>PaymentMethod__c</fullName>
+        <description>The form of payment for this Recurring Donation. NPSP automatically copies the selected value to each related Payment record.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The form of payment for this Recurring Donation. NPSP automatically copies the selected value to each related Payment record.</inlineHelpText>
+        <label>Payment Method</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Credit Card</fullName>
+                    <default>false</default>
+                    <label>Credit Card</label>
+                </value>
+                <value>
+                    <fullName>Check</fullName>
+                    <default>false</default>
+                    <label>Check</label>
+                </value>
+                <value>
+                    <fullName>ACH/EFT</fullName>
+                    <default>false</default>
+                    <label>ACH/EFT</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>RecurringDonation__c</fullName>
+        <description>The Recurring Donation this schedule record is associated with.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The Recurring Donation this schedule record is associated with.</inlineHelpText>
+        <label>Recurring Donation</label>
+        <referenceTo>npe03__Recurring_Donation__c</referenceTo>
+        <relationshipLabel>Recurring Schedules</relationshipLabel>
+        <relationshipName>Recurring_Schedules</relationshipName>
+        <relationshipOrder>0</relationshipOrder>
+        <reparentableMasterDetail>false</reparentableMasterDetail>
+        <trackTrending>false</trackTrending>
+        <type>MasterDetail</type>
+        <writeRequiresMasterRead>false</writeRequiresMasterRead>
+    </fields>
+    <fields>
+        <fullName>StartDate__c</fullName>
+        <description>The starting date for this schedule record.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The starting date for this schedule record.</inlineHelpText>
+        <label>Start Date</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Date</type>
+    </fields>
+    <label>Recurring Schedule</label>
+    <nameField>
+        <displayFormat>RD-Schedule-{0000000000}</displayFormat>
+        <label>Recurring Schedule Num</label>
```

Comment:
Can we use `Recurring Schedule Name`, `Schedule Name` or just `Name` for the record Name label?